doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -25,11 +25,12 @@ Some Key Differences
 
 .. list-table::
 
-   * - In MATLAB®, the basic data type is a multidimensional array of
-       double precision floating point numbers.  Most expressions take such
-       arrays and return such arrays.  Operations on the 2-D instances of
-       these arrays are designed to act more or less like matrix operations
-       in linear algebra.
+   * - In MATLAB®, you can create multidimensional arrays. By default,
+       MATLAB® creates two dimensional arrays of double precision floating point
+       numbers.  Most expressions take such arrays and return arrays.
+       Operations on the 2-D instances of these arrays are designed to act 
+       like matrix operations in linear algebra.
+
      - In NumPy the basic type is a multidimensional ``array``.  Operations
        on these arrays in all dimensionalities including 2D are element-wise
        operations.  One needs to use specific functions for linear algebra
```

Comment:
The way this was worded before was nice, because the sentence had similar structure in both columns. Can you try to bring the two cells into a similar sentence structure?

Reply:
That's a good idea, I'll make make the structure similar in both. The main edit here was to highlight

* Matlab default = 2D arrays
* NumPy default = 0D or 1D arrays

Reply:
I'd say perhaps more relevant is:

* Matlab: 2 or more dimensional arrays, default double
* Numpy: n-dimensional arrays, default varies

Lets see what others think.

Reply:
> I'd say perhaps more relevant is:
> 
> * Matlab: 2 or more dimensional arrays, default double
> * Numpy: n-dimensional arrays, default varies

I like that. In this case, one of the benefits of NumPy over MATLAB is the control over the shapes and types of your arrays. So maybe it would be good to call out that the `array` function defaults to your data types and dimensions. 

I'm thinking about 2 inputs of MATLAB vs Python here:

1. A single integer input:
```matlab
> a=1
> shape(a)
ans =
  1 1
> class(a)
  ans = double
```
vs 
```python
>>> a=np.array(1)
>>> a.shape
  ()
>>> a.dtype
  dtype('int64')
```
2. A list of integers:
```matlab
> a=[1,2,3]
> shape(a)
ans = 
  1 3
> class(a)
  ans = double
```
vs
```python
>>> a=np.array([1,2,3])
>>> a.shape
(3,)
>>> a.dtype
  dtype('int64')
```

Reply:
That definitely looks like something which would be useful to have somewhere on the page.

Perhap the matlab "list of integers" example should use `int8([1, 2, 3])` or whatever the matlab syntax is, for fairness.

Reply:
Maybe this example should go in the table below. The equivalent for matlab signed integers would be `int64([1,2,3])`.